source/tutorials/security/trusty-enabling/trusty-enabling.rst

```diff
@@ -0,0 +1,73 @@
+.. _trusty-enabling:
+
+How to Enable or Disable Trusty for Debugging
+#############################################
+
+`Trusty <https://01.org/trusty-ia>`_ is a mandatory component since Android Oreo dessert onwards. It provides the hardware backed security services such as the keymaster, gatekeeper, crypto etc, which are required for applications such as data storage encryption etc.
+
+How to enable Trusty
+====================
+
+Project |C| supports Trusty. To enable Trusty explicitly, update the config file *device/intel/project-celadon/$<lunch-target>/mixins.spec* in the device config project '*device-androidia*' as follows:
+
+.. code-block:: none
+    :emphasize-lines: 6
+
+    ..
+    [groups]
+    device-specific: celadon
+    treble: true
+    ..
+    trusty: true(ref_target=$<target name>)
+
+The following parameters are valid options within the brackets in the previous configuration setting:
+
+* **ref_target** : ``mandatory``
+    This option defines the reference target device.
+
+    For project |C|, *<target name>* can be any of below values
```

Comment:
Suggested rewrite: 
For project |C|, *<target name>* can be any of the following values: